connector_ecommerce/product.py

```diff
@@ -115,13 +115,13 @@ def _price_changed(self, vals):
         There is no guarantee that's the price actually changed,
         because it depends on the pricelists.
         """
-        type_model = self.env['product.price.type']
-        price_fields = type_model.sale_price_fields()
-        if any(field in price_fields for field in vals):
-            session = ConnectorSession(self.env.cr, self.env.uid,
-                                       context=self.env.context)
-            for prod_id in self.ids:
-                on_product_price_changed.fire(session, self._name, prod_id)
+#         type_model = self.env['product.price.type']
+#         price_fields = type_model.sale_price_fields()
+#         if any(field in price_fields for field in vals):
+        session = ConnectorSession(self.env.cr, self.env.uid,
+                                   context=self.env.context)
+        for prod_id in self.ids:
```

Comment:
It is used by `magentoerpconnect_pricing` to export prices to Magento when they are changed (https://github.com/OCA/connector-magento/blob/abe9a8c480e0e64d120fdf2e50fb23d6e25687cb/magentoerpconnect_pricing/product.py#L131) and perhaps other implementation use it too.